dev/com.ibm.ws.jbatch.cdi/src/com/ibm/ws/jbatch/cdi/BatchCDIInjectionExtension.java

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2013, 2022 International Business Machines Corp.
+ * Copyright 2013 International Business Machines Corp.
```

Comment:
I realise that this was a git revert... but I wonder if the copyright date should now be `2013, 2023`. It is a significant functional change made this year, but it wasn't really introducing _new_ IP. I'm not really sure what date it should be. @jhanders34 what do you think?

Reply:
I general I think many of the copyrights should be updated in this PR to include 2023.  I was ok with the revert of the commit leaving it be 2013 without the 2023, but I can see changing to 2013, 2023 as well.

Reply:
@tevans78 That makes sense to me, I went ahead and added 2023 to those copyright statements in the latest commit.